### ISSUE TITLE
Feature request: Support for CODEBUILD_CLONE_REF in Source of deployment map

### ISSUE THREAD

**Issue by nagmesh** [awslabs/aws-deployment-framework#387]
I would like to have `CODEBUILD_CLONE_REF` as an option on [source output artifact format](https://docs.aws.amazon.com/codepipeline/latest/userguide/action-reference-CodeCommit.html#action-reference-CodeCommit-config) so that I can do some git specific actions in Codebuild

**Comment by StewartW** [comment]
Do you see this just as a property on the code commit source provider? 
```
default_providers:
  source:
    provider: codecommit
    properties:
      output_artifact_format:  CODEBUILD_CLONE_REF | CODE_ZIP
```
Would that work you?

**Comment by nagmesh** [comment]
@StewartW  That would be awesome

**Comment by nagmesh** [comment]
We use a lot of submodules and that would really help. 
Currently, I manually change the pipeline source but if ADF can do it. It would be great!

**Comment by sbkok** [comment]
Thank you for your patience. I am happy to inform you that this feature has been [released as part of v3.2.0](https://github.com/awslabs/aws-deployment-framework/releases/tag/v3.2.0) just now.
I'm hereby closing this issue. Please open a new issue if you are experiencing any issues related to this feature.
